src/gennylib/include/gennylib/v1/PoolManager.hpp

```diff
@@ -129,7 +129,16 @@ class PoolManager {
     /**
      * Obtain a connection or throw if none available.
      *
-     * This is allowed to be called from multiple threads simultaneously.
+     * This connection will be pre-warmed by default, unless "NoPreWarm" is set to true, e.g.:
+     * 
```

Comment:
nit: there are trailing space chars on this line and line 158, please delete them :)

Reply:
Done, thanks!